tests/test_unit/test_transfers/test_file_transfers/test_create_transfer.py

```diff
@@ -10,7 +10,7 @@
 
 
 @pytest.mark.parametrize(
```

Comment:
TODO: maybe we need to refactor test fixtures, there are a lot of annotations over small piece of test code